libnavui-maps/src/main/res/layout/mapbox_camera_mode_button_layout.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    tools:background="@drawable/mapbox_bg_button"
```

Comment:
I don't see `mapbox_bg_button` anywhere. Can we replace it with the correct one?

Reply:
Good catch. Forgot to update this one.